apps/builder/app/builder/features/style-panel/shared/property-name.tsx

```diff
@@ -62,6 +63,24 @@ const PropertyPopoverContent = ({
           {properties.map((property) => {
             const styleValueInfo = style[property];
 
+            if (styleValueInfo?.nextSource) {
+              const { value, styleSourceId } = styleValueInfo.nextSource;
+              const styleSource = styleSources.get(styleSourceId);
+              let name: undefined | string = undefined;
+              if (styleSource?.type === "local") {
+                name = "local style source";
+              }
+              if (styleSource?.type === "token") {
+                name = `"${styleSource.name}" token`;
+              }
+              return (
+                <Text key={property} color="subtle">
+                  The value of {property} is overriden by {toValue(value)} from{" "}
+                  {name}
```

Comment:
this could be better off as a single template literal

Reply:
Probably 🤷